AdlsSdkSamples/Program.cs

```diff
@@ -18,135 +16,102 @@ class Program
         }
-        private static void PerformWriteFlushReadSeek(AdlsClient client)
+        private static void PerformWriteFlushReadSeek(DataLakeFileSystemClient client)
         {
-            // Illustrate stream operations of Adls
-
+            long length;
             string fileName = "/Test/dir1/testFilename1.txt";
 
+            DataLakeFileClient file = client.GetFileClient(fileName);
+
             // Create a file. It creates the parent directories /Test/dir1
-            using (var createStream = client.CreateFile(fileName, IfExists.Overwrite))
+            using (var stream = new MemoryStream())
             {
                 // Write byte-array to stream
                 byte[] writeData = Encoding.UTF8.GetBytes("This is the first line.\n");
-                createStream.Write(writeData, 0, writeData.Length);
+                stream.Write(writeData, 0, writeData.Length);
                 // Flush the data in buffer to server
-                createStream.Flush();
+                stream.Flush();
                 writeData = Encoding.UTF8.GetBytes("This is the second line.\n");
-                createStream.Write(writeData, 0, writeData.Length);
-            } // Closing the stream flushes remaining data
+                stream.Write(writeData, 0, writeData.Length);
+                length = stream.Length;
+                stream.Seek(0, SeekOrigin.Begin);
+                file.Upload(stream, true);
+            }// Closing the stream flushes remaining data
 
             // Append to the file
-            using (var appendStream = client.GetAppendStream(fileName))
+            using (var stream = new MemoryStream())
             {
                 byte[] writeData = Encoding.UTF8.GetBytes("This is the third line.\n");
-                appendStream.Write(writeData, 0, writeData.Length);
+                stream.Write(writeData, 0, writeData.Length);
                 writeData = Encoding.UTF8.GetBytes("This is the fourth line.\n");
-                appendStream.Write(writeData, 0, writeData.Length);
+                stream.Write(writeData, 0, writeData.Length);
+                stream.Seek(0, SeekOrigin.Begin);
+                file.Append(stream, length);
+                file.Flush(length + stream.Length);
             }
 
             // Read the file
-            using (var readStream = client.GetReadStream(fileName))
+            Response<FileDownloadInfo> fileContents = file.Read();
+            using (var readStream =fileContents.Value.Content)
             {
                 byte[] readData = new byte[1024];
 
-                // Seek to offset 24
-                readStream.Seek(24, SeekOrigin.Current);
-
                 // Read 40 bytes at this offset
-                int readBytes = readStream.Read(readData, 0, 40);
+                int readBytes = readStream.Read(readData, 25, 40);
                 Console.WriteLine("Read output of 40 bytes from offset 25: " + Encoding.UTF8.GetString(readData, 0, readBytes));
```

Comment:
We should show off `client.OpenRead()` instead because that's much more appropriate to this sample instead of downloading the entire file.

Reply:
@tg-msft Updated.